Installation.md

```diff
@@ -1,76 +1,152 @@
+
+(This might print some warnings, see the next section where we go into more detail)
+
+Note: In a production context, you shouldn't run a script - hand the `app` object to a WSGI process, as your platform of choice describes.
 Often, that requires a WSGI script. We provide an example WSGI script in [the CI Readme](ci/Readme.md).
 
 
-## Loading data
+### Add your first asset 
+
+Head over to `http://localhost:5000/assets` and add a new asset there.
+
+TODO: [issue 57](https://github.com/SeitaBV/flexmeasures/issues/57) should create a CLI function for this.
+
+Note: You can also use the [`POST /api/v2_0/assets`](https://flexmeasures.readthedocs.io/en/latest/api/v2_0.html#post--api-v2_0-assets) endpoint in the FlexMeasures API to create an asset.
+
+### Add data
+
+You can use the [`POST /api/v2_0/postMeterData`](https://flexmeasures.readthedocs.io/en/latest/api/v2_0.html#post--api-v2_0-postMeterData) endpoint in the FlexMeasures API to send meter data.
+
+TODO: [issue 56](https://github.com/SeitaBV/flexmeasures/issues/56) should create a CLI function for adding a lot of data at once, from a CSV dataset.
+
+Also, you can add forecasts for your meter data with the `db_populate` command, here is an example:
+
+   `flexmeasures db-populate --forecasts --from-date 2020-03-08 --to-date 2020-04-08 --asset-type Asset --asset my-solar-panel `
+
+Note: You can also use the API to send forecast data.
+
+
+## Other settings, for full functionality
+
+### Set mail settings
+
+For FlexMeasures to be able to send email to users (e.g. for resetting passwords), you need an email account which can do that (e.g. GMail). Set the MAIL_* settings in your configuration, see [the Configuration documentation](Configuration.md).
+
+
+### Install an LP solver
+
+For planning balancing actions, the FlexMeasures platform uses a linear program solver. Currently that is the Cbc solver. See the `FLEXMEASURES_LP_SOLVER` config setting if you want to change to a different solver.
+
+Installing Cbc can be done on Unix via:
+
+    apt-get install coinor-cbc
+
+(also available in different popular package managers).
+
+We provide a script for installing from source (without requiring `sudo` rights) in [the CI Readme](ci/Readme.md).
+
+More information (e.g. for installing on Windows) on [the Cbc website](https://projects.coin-or.org/Cbc).
+
+
+
+### Preparing the job queue database and start workers
 
-If you have a SQL Dump file, you can load that:
+To let FlexMeasures queue forecasting and scheduling jobs, install a [Redis](https://redis.io/) server and configure access to it within FlexMeasures' config file (see above). You can find the necessary settings in [the Configuration documentation](Configuration.md).
 
-    psql -U {user_name} -h {host_name} -d {database_name} -f {file_path}
+Then run one worker for each kind of job (in a separate terminal):
 
-Else, you can populate some standard data, most of which comes from files:
+    flexmeasures run_worker --queue forecasting
```

Comment:
Shall I update this command to run-worker (dash instead of underscore), for consistency with other CLI commands? Similarly for clear_queue -> clear-queue

Reply:
Yes. I can also do that together with your other remark(s)